src/core/hook-state.ts

```diff
@@ -6,11 +6,11 @@ export type StateKey<T> = (string | symbol) & { __keyType?: T };
 export class HookState {
 	private _data = new Map<string | symbol, {}>();
 
-	has(key: StateKey<{}>): boolean {
+	has (key: StateKey<{}>): boolean {
 		return this._data.has(key);
 	}
 
-	get<T extends {}>(key: StateKey<T>, defaultValue: T) {
+	get <T extends {}>(key: StateKey<T>, defaultValue: T) {
```

Comment:
This is a (known) bug in the prettier-plugin-space-before-function-paren plugin I still don't know how to fix. The correct formatting should be: `get<T extends {}> (key: StateKey<T>, defaultValue: T)`.

Reply:
It should be fixed in [`v0.0.7`](https://www.npmjs.com/package/prettier-plugin-space-before-function-paren). @robert-j-webb, could you check if it work as expected, please?

Reply:
Can we make sure to pull the latest version of this before merging?